### ISSUE TITLE
Implement DataDictionary Picklists

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/hpi#752]
DataDictionary picklists are available in the HPI admin, but currently do nothing.  See the screencam below:

![dd-picklist](https://cloud.githubusercontent.com/assets/1377228/8959900/eae70f16-35d2-11e5-9a96-61ea6916d077.gif)

I would expect that the picklist config would ask the admin to choose the Object type and then attribute that the picklist should derive from.

**Comment by dgrumieaux** [comment]
Note that the intent for datadictionary picklists in the past has been to utilize the Alfresco-specific constraints that are configured as part of the Alfresco XML object models.

**Comment by gsteimer** [comment]
@dgrumieaux agreed, that was the original intent.  However, you can do the same thing in Documentum's data dictionary, so we should aim to support both.
